content/en/platform/corda/5.0-dev-preview-1/tutorials/building-cordapp/c5-basic-cordapp-intro.md

```diff
@@ -35,7 +35,7 @@ This example showcases a delivery versus payment (DvP) scenario. You are buildin
 
 There are two parties involved in this transaction:
 
-- The space travel company - Mars Express.
+- A space travel company - Mars Express.
 - A customer - let's call him Peter.
```

Comment:
Lets call them Petunia, more women in tech examples :)

Reply:
We might stick with Peter for this tutorial, then use Petunia in the next one we create.